cypress/fixtures/articles_index.json

```diff
@@ -2,23 +2,27 @@
   "articles": [
     {
       "id": 1,
-      "title": "Manchester City loves e-scooters",
-      "lead": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut odit eius facere tempora expedita, ratione itaque quidem sequi totam nulla? Eaque possimus sit a officia."
+      "title": "Ny studie: Möjligt att utplåna fattigdomen och samtidigt rädda klimatet",
+      "lead": "Den ”omöjliga ekvationen” går att lösa.",
+      "content": "Det är fullt möjligt att rädda jordens klimat och utplåna fattigdomen på samma gång, hävdar brittiska forskare. Och detta utan att kasta oss tillbaka till medeltiden."
```

Comment:
Why have you added content to the index fixture?

Reply:
Our mistake, we only added to see how an article looked like in a semantic Card and then we forgot to remove.